scripts/utils/fetchWithTimeout.ts

```diff
@@ -0,0 +1,20 @@
+import fetch, { RequestInit, Response } from 'node-fetch'
```

Comment:
is there a reason to use fetch vs got or axios?

Reply:
Not as far as I know, other than the fact that it's already being used in this repo in the [other script](https://github.com/mobilestack-xyz/funding-layer/blob/main/scripts/protocol-filters/beefy.ts#L1) that's checked in.